src/i18n/translations/en.yml

```diff
@@ -5,6 +5,11 @@ Notfound: Notfound
 Published: Published
 Unpublished: Unpublished
 not set: not set
+Open timeline: 'Open timeline'
+Close timeline: 'Close timeline'
+Playback timeline: 'Playback timeline'
+Play timeline: 'Play timeline'
+ellipse: 'ellipse'
```

Comment:
I am not familiar with screen readers and how they are used enough to know, but are these detailed enough? Will aria labels typically be short?

Reply:
Typically `aria-label` should be set text same as able-bodied person see usually.
If UI has button like `<button><Icon /></button>`, we can understand what button means from icon but visually impaired person can not see icon so this person can not understand about this button. So we need to describe about Icon by `aria-label`.
If we set aria-label like `Open timeline`, screen reader says `"Open timeline button is clicked"` by clicking `<button><Icon /></button>`.

Reply:
Ah ok! So these terms will be used in a sentence by the screenreader. Ok!